src/cell/main.c

```diff
@@ -118,6 +239,18 @@ parse_command_line(int argc, char* argv[])
 				print_banner(true, true);
 				return false;
 			}
+			else if (strcmp(argv[i], "--init") == 0) {
```

Comment:
My preference is for the command-line syntax for this to be:
```
cell init
```
but that will require a change to the commandline parser.  `--init` will do for now.

Reply:
It doesn't use that syntax because it would be inconsistent with the rest of the command line options.

Reply:
I’m aware; but those others are simply options, this is logically a separate command entirely.  Kind of like how Git has its commands (`git commit`) as a separate concept from options (`git commit --amend`).

Logically cleaning should be `cell clean` for the same reason.